src/components/Doctors/Doctors.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from "react";
+
+import DoctorList from "components/Doctors/DoctorList";
+import Loader from "components/Loader";
+import { getDoctors } from "api/doctors";
+
+const Doctors = () => {
+  const [count, setCount] = useState(0);
+  const [nextPage, setNextPage] = useState(0);
+  const [rows, setRows] = useState(null);
+  const [error, setError] = useState(null);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const fetch = async () => {
+      const response = await getDoctors();
```

Comment:
you are missing try-catch. What if the request fails? The user will not see an error message.